lib/iris/analysis/maths.py

```diff
@@ -610,14 +616,14 @@ def _broadcast_cube_coord_data(cube, other, operation_noun, dim=None):
             except iris.exceptions.CoordinateNotFoundError:
                 raise ValueError("Could not determine dimension for %s. "
                                  "Use %s(cube, coord, dim=dim)"
-                                 % (operation_noun, operation_noun))
+                                 % (operation_name, operation_name))
 
     if other.ndim != 1:
         raise iris.exceptions.CoordinateMultiDimError(other)
 
     if other.has_bounds():
         warnings.warn('%s by a bounded coordinate not well defined, ignoring '
-                      'bounds.' % operation_noun)
+                      'bounds.' % operation_name)
```

Comment:
I agree that it doesn't read very well as it stands - I'll have a go at improving it.